tools/src/code-review/reviewers/checkMissingChangelogs.ts

```diff
@@ -22,18 +22,22 @@ export default async function ({ pullRequest, diff }: ReviewInput): Promise<Revi
   }
```

Comment:
I meant to move the condition to this `if` ☝️ 😅

Reply:
Yeah, I got it, but for some reason I though we map the PRs not packages in the first loop. 😅 

TBH, we can even bail out earlier, when there is no `head` to avoid the iterating over package, what do you think?